packages/sdk/src/evm/core/classes/transactions.ts

```diff
@@ -223,6 +223,25 @@ export class Transaction<TResult = TransactionResult> {
     return this.contract.interface.encodeFunctionData(this.method, this.args);
   }
 
+  /**
+   * Multiply the gas limit by a factor
+   *
+   * @example
+   * ```js
+   * // Multiple by a factor of 1.2 (increase by 20%)
+   * await tx.multiplyGasLimit(1.2)
+   * ```
+   */
+  async multiplyGasLimit(factor: number) {
```

Comment:
it feels weird to make this async - we should set the % on the txn class and then do the operation with it applied (when we already estimate gas), otherwise I do this -> wait 5min -> then send later and it'll be off?

Reply:
True - changed it to synchronous